tests/console/sle15_workarounds.pm

```diff
@@ -23,6 +23,9 @@ use utils qw(zypper_call sle_version_at_least);
 
 sub run {
     my $self = shift;
+    # Stop packagekit
+    script_run "systemctl mask packagekit.service";
```

Comment:
Please, use systemctl subroutine from lib/utils.pm

Reply:
for reference:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L812
;)

Reply:
Now which one shall I use? "systemctl" L147 or "pkcon_quit" L812 ?

Reply:
I don't know to which @rwx788 was referring, but I would prefer `pkcon_quit` ;)

Reply:
I was not aware of pkcon_quit, yes, it's better!